Dockerfile

```diff
@@ -211,3 +211,7 @@ FROM base AS final-organisation-app
 WORKDIR /app
 COPY --from=publish-organisation-app /app/publish .
 ENTRYPOINT ["dotnet", "CO.CDP.OrganisationApp.dll"]
+
+FROM localstack/localstack:3.5 AS localstack
+COPY --chown=localstack ./localstack-init/create-queues.sh /etc/localstack/init/ready.d/init-aws.sh
+RUN chmod +x /etc/localstack/init/ready.d/init-aws.sh
```

Comment:
So far, we've only been using this Dockerfile for production dotnet services. I'd like to keep it like that. Localstack is not only not a production service, but entirely unrelated to our dotnet services (doesn't share any build steps). 

I think in this case it's not even necessary to create a dedicated image. Just mount the ready.d volume from `./docker/localstack/ready.d`.